src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +208,230 @@
+                                                <Run Text="{Binding LastSettingsBackupDate, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="0" Grid.Column="2" BorderThickness="1,0,0,0" BorderBrush="Black">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_LastSettingsRestoreDate" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="0" Grid.Column="3">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding LastSettingsRestoreDate, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+
+                                <Border Grid.Row="2" Grid.Column="0">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_LastSettingsBackupSource" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="1">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding LastSettingsBackupSource, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="2" BorderThickness="1, 0, 0, 0" BorderBrush="Black">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_CurrentSettingMatchStatus" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="3">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding CurrentSettingMatchText, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                            </Grid>
+
+
+
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
+                </controls:SettingsGroup>
+
+
+                <InfoBar x:Uid="General_SettingsBackupMessageResults"
+                                  IsClosable="False"
+                                  Severity="{Binding BackupRestoreMessageSeverity, Converter={StaticResource StringToInfoBarSeverityConverter}}"                         
```

Comment:
@jefflord 
When do we have whuch severity?

Reply:
A config error
![image](https://user-images.githubusercontent.com/4396667/192775744-4ad9d9d0-f784-4e68-9c6b-1e736a22eefa.png)

An operation error
![image](https://user-images.githubusercontent.com/4396667/192775868-c2bac865-25cc-4ca6-bb19-069c2303dfef.png)

Task complete
![image](https://user-images.githubusercontent.com/4396667/192776087-81be21dc-9fc3-4f42-8b11-62de2a43824e.png)

Task complete, but nothing to do
![image](https://user-images.githubusercontent.com/4396667/192777331-376d7eb7-34fd-4f4a-a5f2-6e8209805ace.png)

Task complete, but nothing to do, here we have a warning because we need a backup to restore.
![image](https://user-images.githubusercontent.com/4396667/192776648-55ea9a19-d935-454e-9a25-99f9100a704e.png)